test/e2e/validation_test.go

```diff
@@ -503,3 +503,65 @@ func (tc *testContext) validateUpgradeableCondition(expected meta.ConditionStatu
 	}
 	return nil
 }
+
+// runWindowsJob creates a Windows Server Job with the given `name` and execution `command`. In addition, check for
+// errors based on `expectError` and is able to delete the job if specified by `deleteJob`
+func runWindowsJob(name, command string, expectError, deleteJob bool,
```

Comment:
Instead of passing in testCtx please make this a method of it

Reply:
I am thinking that what this function is trying to achieve would be better done by changing https://github.com/openshift/windows-machine-config-operator/blob/ae1fa6c042dbeec3078aae68eba11c3507999caf/test/e2e/network_test.go#L572 to take a powershell command, as the only callsite of it is also using `testCtx.getPowerShellExe(), "-Command",`

Reply:
+1

Reply:
Makes sense, will address it in a follow-up PR.

Reply:
You are introducing this code in this PR. I don't see why this needs a follow up PR. Please address this in the current PR.

Reply:
Addressed in https://github.com/openshift/windows-machine-config-operator/pull/777/commits/54cbfc35ed8bc73c2475443690d3d3e9848dcb8a